f5/bigip/shared/file_transfer.py

```diff
@@ -17,6 +17,7 @@
 #
 
 import os
+from StringIO import StringIO
```

Comment:
Could attempt cPython's StringIO, as shown in https://pymotw.com/2/StringIO/#example

In case the files are yuuge.